apps/openmw/mwmechanics/aiwander.cpp

```diff
@@ -227,8 +239,20 @@ namespace MWMechanics
         }
 
         // Actor becomes stationary - see above URL's for previous research
-        if(mAllowedNodes.empty())
+        // If not an NPC and no pathgrid is available, randomly idle or wander around near spawn point
+        if(mAllowedNodes.empty() && !actor.getBase()->mClass->isNpc() && !storage.mIsWanderingManually) {
```

Comment:
@scrawl I added code to handle NPCs also. Unfortunately, I don't know of any test cases for NPCs that are scripted to wander around an area without pathgrids (I removed pathgrids from Seyda Neen but even in Vanilla all the units just stand around); do you know of a good test case for that?